public/styles/style.scss

```diff
@@ -126,11 +126,11 @@ article {
 .button-with-icon {
     font-size: 0.7em;
 
-    width: 3em;
+    min-width: 3em;
```

Comment:
```suggestion
```
you can delete this CSS property. it works great even without it! (I didn't know it)
![image](https://github.com/user-attachments/assets/28ee450c-59c5-45ba-a5ad-be20585a4e8e)